saleor/app/migrations/0028_set_app_identifier.py

```diff
@@ -0,0 +1,45 @@
+from django.db import migrations
+from django.db.models import F, QuerySet
+
+from ..models import App
+
+BATCH_SIZE = 1000
```

Comment:
The batch is here just in case, as we expect that clients have ~10 apps.
The migration also changed, for that case I will leave the batch as for now.